### PR TITLE
ref #692: Fix active icon list display

### PR DESCRIPTION
**Pull request by UlrichKu**
| Q             | A
| ------------- | ---
| Branch        | 7.0.x
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#692
| License       | MIT

### CODE REVIEW

File: src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1615360383.inc.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<h1>Build #1615360383</h1>
+<h2>Date: 2021-03-10</h2>
+<div class="changelog">
+    - #692: Fix active icon list display
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('cms_tbl_display_list_fields', 'de')
+  ->setFields([
+      'callback_fnc' => 'gcf_GetAciveIcon',
```
Comment (bestform) [inline src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1615360383.inc.php:10]: Shouldn't that be `getActiveIcon`?

**Review by bestform — CHANGES_REQUESTED** [review]

File: src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1615360383.inc.php (hunk 1 above)
Comment (pixeljunkie) [reply src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1615360383.inc.php:10]: Nope. The prefix has matching reasons. gcf_ = global callback function (core), ccf_ = userspace

Comment (pixeljunkie) [reply src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1615360383.inc.php:10]: Thinking about this... this file based function repository is kind of deprecated. It is possible to add callback functions to Tdb record classes. If there is a getActiveIcon method this is the one, it should reference.

Comment (bestform) [reply src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1615360383.inc.php:10]: I was not addressing the prefix, but the missing `t` in `Acive` :)

Comment (pixeljunkie) [reply src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1615360383.inc.php:10]: Oh... never mind... :-)

Comment (UlrichKu) [reply src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1615360383.inc.php:10]: getAciveIcon is how it is named...

Comment (bestform) [reply src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1615360383.inc.php:10]: Oh my...

**Pull request merged**